x/upgrade/keeper/keeper.go

```diff
@@ -36,6 +36,7 @@ type Keeper struct {
 	upgradeHandlers    map[string]types.UpgradeHandler // map of plan name to upgrade handler
 	versionSetter      xp.ProtocolVersionSetter        // implements setting the protocol version field on BaseApp
 	downgradeVerified  bool                            // tells if we've already sanity checked that this binary version isn't being used against an old state.
+	authority          sdk.AccAddress				   // the address capable of executing and cancelling an upgrade. Usually the gov module account
```

Comment:
```suggestion
	authority          string				   // the address capable of executing and cancelling an upgrade. Usually the gov module account
```

I would prefer string here, so that we don't depend on the global bech32 singleton.

Also, any opinions on using a `config Config` here? ref: https://github.com/cosmos/cosmos-sdk/pull/11116#discussion_r816637995

Reply:
I've changed it to a string. I don't think this needs a config because I don't think this should have a default.